rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -232,131 +233,51 @@ def build(self, view_requirements: ViewRequirementsDict) -> SampleBatch:
                     _to_float_np_array(d) for d in self.buffers[data_col]
                 ]
 
-            # Range of indices on time-axis, e.g. "-50:-1". Together with
-            # the `batch_repeat_value`, this determines the data produced.
-            # Example:
-            #  batch_repeat_value=10, shift_from=-3, shift_to=-1
-            #  buffer=[-3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
-            #  resulting data=[[-3, -2, -1], [7, 8, 9]]
-            #  Range of 3 consecutive items repeats every 10 timesteps.
-            if view_req.shift_from is not None:
-                # Batch repeat value > 1: Only repeat the shift_from/to range
-                # every n timesteps.
-                if view_req.batch_repeat_value > 1:
-                    count = int(
-                        math.ceil(
-                            (len(np_data[data_col][0]) - self.shift_before)
-                            / view_req.batch_repeat_value
-                        )
+            assert (
```

Comment:
can we move this out of simple_list_collector and into a util.py file under evaluation/ maybe?
or you want to do this with next pr?

Reply:
next PR. I'll use agent_collector directly. I already have that going.